frontend/packages/component-library/src/cms/heroBanner/heroBanner.module.scss

```
@@ -0,0 +1,93 @@
+@use '@code-dot-org/component-library-styles/colors.scss';
+@use '@code-dot-org/component-library-styles/variables' as variables;
+@use '@code-dot-org/component-library-styles/typography.module' as typography;
+
+$mobileBreakpoint: 1024px;
+
+.heroBannerWrapper {
+  width: 100%;
+  background: var(--background-neutral-tertiary);
+  background-size: cover;
+
+  &.heroBanner-withWideText {
+    .heroBannerDescriptiveSection {
+      flex: 0 0 65%;
+    }
+
+    .heroBannerVisualSection {
+      flex: 0 0 30%;
+    }
+  }
+}
+
+.heroBannerContainer {
+  display: flex;
+  max-width: 60rem;
+  margin: 0 auto;
+  align-items: center;
+  gap: 4.375rem;
+  padding: 4rem 2rem;
+
+  @media (max-width: $mobileBreakpoint) {
+    flex-direction: column;
+    padding: 2rem 1rem;
```

Update to match the Figma:
```suggestion
    padding-inline: 1.5rem;
```

---

Thank you for sotting this!
Didn't have figma designs for mobile when implemented it